src/components/Article.jsx

```diff
@@ -1 +1,19 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = props => (
```

Comment:
let's destructure props 
```suggestion
export const Article = ({ title, date, text }) => (
```